src/dlgProfilePreferences.cpp

```diff
@@ -2072,12 +2072,14 @@ void dlgProfilePreferences::loadMap()
         return;
     }
 
-    QString fileName = QFileDialog::getOpenFileName(
-                           this,
-                           tr("Load Mudlet map"),
-                           mapSaveLoadDirectory(pHost),
-                           tr("Mudlet map (*.dat *.json);;Xml map data (*.xml);;Any file (*)",
-                              "Do not change extensions (in braces) as they are used programmatically"));
+    auto loadExtensions(QStringList()
+        <<   tr("Mudlet binary (*.dat)", "Do not change extensions (in braces) as they are used programmatically")
+        <<   tr("Mudlet JSON map (*.json)", "Do not change extensions (in braces) as they are used programmatically")
+        <<   tr("Xml map data (*.xml)", "Do not change extensions (in braces) as they are used programmatically")
```

Comment:
Given that only a specific format of XML data can be read as a Map perhaps this should be a bit more qualified? 
```suggestion
        <<   tr("Mudlet XML map data (*.xml)", "Do not change extensions (in braces) as they are used programmatically")
```

Reply:
How about
- Mudlet binary map
- Mudlet JSON map
- Mudlet XML map

Reply:
Is the XML map data different from the xml maps being sent by, say, the IRE games? If it's the same one isn't there a name for that standard already?

Reply:
> Is the XML map data different from the xml maps being sent by, say, the IRE games? If it's the same one isn't there a name for that standard already?

Well, not really, we do not completely use every aspect of the IRE games' data but the importer gets the data that we need to build a Mudlet map.

\<aside>For instance we do not make full use of the "Environment" data they provide - but we don't currently accommodate all of what it provides for each Environment ID:
* "Name" - not used
* "Color" - one of the ANSI 16-color values - inserted into the user **in**accessible `(QMap<int, int>) TMap::mEnvColors` member - where the key is the Id and the value is the Color
* "HTML" -  a 24-bit colour, not used

\</aside>

Reply:
> How about
>
>* Mudlet binary map
>* Mudlet JSON map
>* Mudlet XML map

Well that was part of what I am suggesting - which was effectively those three contained within two others:
* Any Mudlet map type

and:
* Any file

The latter in case someone has a binary map without the specific `.dat` extension and the former to just consider files that appear to be what can be read as a map.